libraries/AP_Mission/AP_Mission.h

```diff
@@ -634,6 +634,9 @@ class AP_Mission
 
     // returns true if the mission has a terrain relative mission item
     bool contains_terrain_alt_items(void);
+    
+    // returns true if the mission cmd has a location
+    static bool cmd_has_location(const uint16_t command);
```

Comment:
we can just make cmd_has_location be !stored_in_location()
that needs to be made public

Reply:
done